src/bundle/construct-manifest.ts

```diff
@@ -39,7 +39,8 @@ ${routes
         path.join(options.routesDirectory, relativePath)
       )}"`
   )
-  .join("\n")}
+  .join("\n")
+  .replace(/\\/g, "/")}
```

Comment:
```suggestion
  // pathfix for windows (esbuild always uses windows backslash paths)
  .replace(/\\/g, "/")}
```

Reply:
@seveibar typo, you meant to say it always uses unix forwardslash

Reply:
Woops